### ISSUE TITLE
Unable to focus on headerview in Talkback mode.

### ISSUE THREAD

**Issue by vr49** [edubarr/header-decor#77]
Hi, I am unable to focus on header view in talkback mode. 

Can anyone please help me on how to focus on the header view ? 

I have already tried setting
focusable: true
focusableInTouchMode: true 
importantForAccessibility: yes. 
and also tried firing the accessibility event, but neither of them work.

**Comment by yuvalldb** [comment]
Hi, did you manage to solve this issue?
